grails-app/views/order/_actions.gsp

```diff
@@ -127,20 +129,9 @@
 								&nbsp;${warehouse.message(code: 'order.deleteOrder.label')}
 							</g:link>
 						</div>
-					</g:supports>
-				</g:if>
-				<g:if test="${orderInstance?.orderType == PUTAWAY_ORDER && orderInstance?.status != OrderStatus.COMPLETED}">
```

Comment:
@kchelstowski why this part was removed?

Reply:
I’m sorry, it was probably a missclick while refactoring if’s structure to look cleaner and by mistake I probably deleted this. Will add it back asap